### PR TITLE
Add lockfile from 8.15 branch

### PR DESCRIPTION
**Pull request by robbavey**

### CODE REVIEW

**Review by mashhurs — APPROVED** [review]
lgtm!

File: Gemfile.jruby-3.1.lock.release [hunk 1]
```diff
@@ -0,0 +1,1012 @@
...
+    march_hare (4.6.0-java)
+    memoizable (0.4.2)
+      thread_safe (~> 0.3, >= 0.3.1)
+    method_source (1.1.0)
+    metriks (0.9.9.8)
+      atomic (~> 1.0)
+      avl_tree (~> 1.2.0)
+      hitimes (~> 1.1)
+    mini_mime (1.1.5)
+    minitar (0.9)
```
Comment (jsvd) [inline Gemfile.jruby-3.1.lock.release:722]:
```suggestion
    minitar (1.0.2)
```

File: Gemfile.jruby-3.1.lock.release [hunk 2]
```diff
@@ -0,0 +1,1012 @@
...
+      clamp (~> 1)
+      concurrent-ruby (~> 1, < 1.1.10)
+      down (~> 5.2.0)
+      elasticsearch (~> 7)
+      filesize (~> 0.2)
+      gems (~> 1)
+      i18n (~> 1)
+      jrjackson (= 0.4.20)
+      manticore (~> 0.6)
+      minitar (~> 0.8)
```
Comment (jsvd) [inline Gemfile.jruby-3.1.lock.release:20]:
```suggestion
      minitar (~> 1)
```

File: Gemfile.jruby-3.1.lock.release [hunk 3]
```diff
@@ -0,0 +1,1012 @@
...
+  logstash-output-http
+  logstash-output-lumberjack
+  logstash-output-nagios
+  logstash-output-null
+  logstash-output-pipe
+  logstash-output-redis
+  logstash-output-stdout
+  logstash-output-tcp
+  logstash-output-udp
+  logstash-output-webhdfs
```
Comment (jsvd) [inline Gemfile.jruby-3.1.lock.release:991]:
```suggestion
  logstash-output-webhdfs
  minitar (~> 1)
```

**Review by jsvd — COMMENTED** [review]
added minitar 1.x pinning

**Review by jsvd — APPROVED** [review]
Actually nvm, I just noticed it's the vanilla lock file from 8.15.3, so let's merge this now and bump the versions afterward. LGTM

**Pull request merged**